requirements.txt

```diff
@@ -3,10 +3,14 @@ requests>=2.0,<3.0dev
 units
 pytz
 
-# testing
+# Testing
 pytest
 responses
 
+# Package build
+build
+twine
```

Comment:
we do not need these two here. IMO.

we'd better just install them in CI

Reply:
totally. I wonder if we just have a small requirements-dev file that we install on builds that need those two packages? that might be an easy way to manage it given i do have to install those on 2 builds now. Some might want to build the wheel as a test locally.  (well i have been doing that)

Reply:
of course,  requirements-dev is fine for me